### PR TITLE
fix: UI mobile stretched beyond screen size [4163]

### PR DESCRIPTION
**Pull request by FatahChan**
Fixed issue 4163: Content creation UI on mobile is starched beyond the screen size by change the style from flex-1 to w-full, then added dynamic top padding so it doesn't conflict with hamburger menu

### CODE REVIEW

**Comment by FatahChan** [comment]
@jeffsee55 

hello, 
This is my first time contributing to open source.

I was wondering if  I am missing a step to get a review, checked the contribution.md, I didn't see anything about the PR process.

Sorry if there is something obvious that I missed

Thanks in advance

File: .changeset/strange-shoes-relate.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'tinacms': patch
+---
+
+Fixed Content creation UI on mobile is starched beyond the screen size by change the style from flex-1 to w-full, then added dynamic top padding so it doesn't conflict with hamburger menu
```
Comment (kldavis4) [inline .changeset/strange-shoes-relate.md:5]:
```suggestion
Fix for issue where content creation UI on mobile is stretched beyond the screen size by changing the style from flex-1 to w-full and then adding dynamic top padding so it doesn't conflict with hamburger menu
```

**Comment by bojanvidanovic** [comment]
Can this get merged? @kldavis4

**Review by BrookJeynes — APPROVED** [review]

**Comment by BrookJeynes** [comment]
Hey @FatahChan, the change looks good.

For future reference, could you please include a screenshot when you make style updates as it helps with visualising the changes  🙏

I think the dynamic top padding might not be necessary, so I've gone ahead and removed it. Was there something this was fixing?
 
Merging as this looks good though, great work!

Thanks,

**Pull request merged**